java/code/src/com/redhat/rhn/frontend/action/help/EulaAction.java

```diff
@@ -36,7 +37,7 @@
  * EULA action page.
  */
 public class EulaAction extends org.apache.struts.action.Action {
-    private static final File EULA_PATH = new File("/srv/www/htdocs/help/eula.html");
+    private static final File EULA_PATH = new File(Config.get().getString("documentroot") + "/help/eula.html");
```

Comment:
I think we should have one of two options:
- pass a default value, to not force it to be defined on the configuration files
- Add a wrapper method to `ConfigDefaults` to return the document root, and that method deals with the defaults.

Reply:
@rjmateus Can you please elaborate your concern/suggestion? Not sure I quite understand it.
"documentroot" is already defined/available in the rhn config. So nothing additional would be required.
Or are you worried that "documentroot" will not exist?

Reply:
My concern is that: what if the property doesn't exist or where removed.
I think we can try to prevent this situation by defining a default value at Java Side.